rust/app/src/handlers/signing.rs

```diff
@@ -132,29 +190,34 @@ impl Viewable for BlindSignUi {
         page: u8,
     ) -> Result<u8, ViewError> {
         if let 0 = item_n {
-            let title_content = bolos::PIC::new(b"Blind Sign\x00").into_inner();
-
+            let title_content = pic_str!(b"Operation");
             title[..title_content.len()].copy_from_slice(title_content);
 
-            let m_len = message.len() - 1; //null byte terminator
-            if m_len <= Sign::SIGN_HASH_SIZE * 2 {
-                let chunk = self
-                    .hash
-                    .chunks(m_len / 2) //divide in non-overlapping chunks
-                    .nth(page as usize) //get the nth chunk
-                    .ok_or(ViewError::Unknown)?;
-
-                hex::encode_to_slice(chunk, &mut message[..chunk.len() * 2])
-                    .map_err(|_| ViewError::Unknown)?;
-                message[chunk.len() * 2] = 0; //null terminate
-
-                let n_pages = (Sign::SIGN_HASH_SIZE * 2) / m_len;
-                Ok(1 + n_pages as u8)
-            } else {
-                hex::encode_to_slice(&self.hash[..], &mut message[..Sign::SIGN_HASH_SIZE * 2])
-                    .map_err(|_| ViewError::Unknown)?;
-                message[Sign::SIGN_HASH_SIZE * 2] = 0; //null terminate
-                Ok(1)
+            let mut mex = [0; 51];
```

Comment:
It could, probably an associated constant to the `Operation` struct, but since this "pattern" is used somewhat often I thought maybe we should refactor it so that `base58_branch` returns the array instead of asking for a mutable reference.

By this pattern I mean passing a mutable array of bytes to the function that encodes into a base58 string

Reply:
Went with everything returning an array, since most of these things is single use anyways, and we also store the whole string in memory so we can chunk it up when writing to the ui